app.go

```diff
@@ -438,3 +449,81 @@ func executeCancelSell(cmd command) bool {
 
 	return true
 }
+
+func executeSetBuyAmount(cmd command) bool{
+	account := accountStore.GetAccount(cmd.UserID)
+
+	if account == nil {
+		log.Infof("User %s does not have an account", cmd.UserID)
+		return false
+	}
+
+	amount, err := currency.NewFromString(cmd.Args[1])
+	if err != nil {
+		// Bail on parse failure
```

Comment:
Comment does not add value. Why are we failing? If we know, we should comment. If not, figure out why and comment.